pkg/deploys/fuse/deployer.go

```diff
@@ -104,6 +94,57 @@ func (fd *FuseDeployer) Deploy(req *brokerapi.ProvisionRequest, async bool) (*br
 	}, nil
 }
 
+func (fd *FuseDeployer) createOperatorResources(namespace string, client client.Client) error {
+	resourcesUrl, exists := os.LookupEnv("FUSE_OPERATOR_RESOURCES_URL")
+	if !exists {
+		return errors.New("FUSE_OPERATOR_RESOURCES_URL environment variable is not set")
+	}
+	glog.Printf("Operator resources url = %v", resourcesUrl)
+
+	var httpClient http.Client
+	resp, err := httpClient.Get(resourcesUrl)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
+	var resources []runtime.Object
+	dec := yamlv2.NewDecoder(resp.Body)
+	for {
```

Comment:
is this loop like this as there are multiple yaml documents in the single file?

Reply:
Might read a little clearer if it was something like

```
var yamlErr error
for yamlErr != io.EOF{

}
```
small thing not big deal

Reply:
Yes its a multi document yaml file